src/components/kytos/misc/Menubar.vue

```diff
@@ -14,6 +14,11 @@
                :icon="component.icon"
                :tooltip="component.tooltip"
                @click.native="setItem(index+1)"/>
+     <k-button v-for="(component, index) in separateComponents"
```

Comment:
@HeriLFIU, I wonder if we could still use the existing `components` instead of adding this new `separateComponents`. I think we can make `KytosStatusMenu` as an official part here too, wdyt? Let me know if you had any other reason what I'm not following here. Also in the `components` it would render before the dynamic NApps ui, so it'd be great too to have it right after the `mapbox-settings`

Reply:
It was giving me a lot of issues, but I just realized its because I forgot to remove the component itself from `APP.vue`, so it would never go away. I just added it to the toolbar and it seems to be working fine now.

Reply:
Now for the key bindings, since its part of the menu bar now, I can just probably set active item to 2.

Reply:
@viniarck actually since its part of the menu bar now, should we have some keybind system where:

The first element could be ctrl+shift+1 then ctrl+shift+2 and so on.

Reply:
because if the menu bar changes, the hot key that was set for active item 2 might also change

Reply:
Interesting, if it's behaving as expected now and having it on `ctrl+shift+2` would be cool. 

Great solution.